endorsement/storeapi.proto

```diff
@@ -43,6 +43,76 @@ message OpenArgs {
 
 message CloseArgs {
 }
+message SwComponentID {
+    common.TokenFormat type = 1;
+    google.protobuf.Struct id = 2;
+}
+// SwComponentInfo defines the structure of a single Software Component 
+// identified by a SoftwareComponent Identifer.
+message SwComponentInfo {
+    SwComponentID comp_id = 1;
+    google.protobuf.Struct comp_meta = 2; 
+}
+
+message AddSwComponentArgs {
+    repeated SwComponentInfo info = 1;
+}
+
+message GetSwComponentArgs {
+    SwComponentID id = 1;
+}
+
+message GetSwComponentRsp {
+    SwComponentInfo sw_component = 1;
+    int32 error_value = 2;
+    string error_detail = 3;
+}
+
+enum TaType {
+    UNSPECIFIED = 0;
+    KEY = 1;
+    CERTIFICATE = 2;
+    INVALID = 255;
+}
+
+message TrustAnchorID {
+    common.TokenFormat type = 1;
+
+    // Trust Anchor ID assumes a form based on the type of TokenFormat
+    google.protobuf.Struct id = 2;
+}
+
+message TrustAnchorValue {
+    TaType type = 1;
+
+    // The value is the actual trust anchor. It is a map with
+    // key-value pairs of TA. What part comprise of a trust anchor value 
+    // is defined by the type.
+    google.protobuf.Struct value = 2;
+}
+
+// Trust Anchor Info holds information about a Trust Anchor
+// A Trust Anchor is uniquely identified by a Trust Anchor ID, while
+// the actual Trust Anchor is conveyed via TrustAnchorValue
```

Comment:
I don't think this comment is adding any useful information on top of what is already a very clear struct layout. Suggest removing it.